.github/workflows/CI.yml

```diff
@@ -0,0 +1,64 @@
+name: CI
+
+on: [push]
+
+jobs:
+  test:
```

Comment:
Can jobs be given names like `Test`?

Reply:
It would look prettier here:

![image](https://user-images.githubusercontent.com/157195/69670011-97e8bc00-1058-11ea-8217-f6a6018eccaf.png)

Reply:
I think so

Reply:
Mind trying?

Reply:
Worked! <img width="330" alt="Screen Shot 2019-11-26 at 2 36 20 PM" src="https://user-images.githubusercontent.com/6101761/69670728-26aa0880-105a-11ea-934b-9be23ce52f2a.png">